lib/workers/repository/config-migration/pr/index.ts

```diff
@@ -45,33 +46,12 @@ ${
       `JSON5 config file migrated! All comments & trailing commas were removed.`
     : ''
 }
----
-{{#if hasWarningsErrors}}
-{{{warnings}}}
-{{{errors}}}
-{{else}}
-#### Migration completed successfully, No errors or warnings found.
-{{/if}}
----
```

Comment:
are we sure we don't want those?

Reply:
I am not sure they will be related to the config migration.